### ISSUE TITLE
Persoonal Performance Rationale 

### ISSUE THREAD

**Issue by oryn** [oryn/oryn.ca#7]
###  Personal Performance Rationale

When starting this entire assignment I was faced with having to honest about my level of coding skills and what I can and cannot do. I designed a website that fit my skills, in turn the website ended up being quit simple and fairly basic. Although I have very little patterns and not many moving parts I gave a good space for my work to live and I am proud of what I was able to complete based on my ability. 
I think where I went wrong was that I made it a bit too simple. But I think of what I would design if I was at a higher skill level and I think I would still keep it simple and basic in design. That being said I did run into problems during the process I ended up compromising the overall design for the lack of my coding ability. I think next time I will try to make “simple” and “basic” a bit more interesting, by that I mean introducing a level of attraction rather than hanging it on my artwork.   
In conclusion, the website is usable and conveys the message. This is my work and here’s how to contact me as well as meeting all the required work. I know is simple but I am proud of it. 

@thomasjbradley
